src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
```

Comment:
Method hide should change hidden property to the opposite value, not always to the true.

Reply:
In a third point of checklist. "this function should just hide but not toggle". But i changed it.

Reply:
Good comment 👍  Truly checklist disagrees with the task description. 
I will keep that in mind 👌